src/igvfd/schemas/tissue.json

```diff
@@ -116,8 +116,8 @@
         },
         "preservation_method": {
             "title": "Preservation Method",
-            "description": "The method by which the tissue was preserved: cryopreservation (slow-freeze) or flash-freezing.",
-            "comment": "Indicate the tissue preservation_method if known and may be/become an experimental variable to track.",
+            "description": "The method by which the tissue/organ was preserved: cryopreservation (slow-freeze) or flash-freezing.",
+            "comment": "Indicate the tissue/organ preservation_method if known and may be/become an experimental variable to track.",
             "type": "string",
             "enum": [
                 "cryopreservation",
```

Comment:
"pooled_from": {
            "items": {
                "title": "Tissue Pooled From",

Should have its title adjusted